docs/wiki/setup-google-cloud-account.md

```diff
@@ -0,0 +1,35 @@
+# Set up a Google Cloud account
+
+
+## Create a Google account
+
+Create an account at [Google Cloud](https://cloud.google.com).
+
+When you have signed in, go to the [Getting Started](https://console.cloud.google.com/getting-started) page. Set your country and agree to the Terms of Service, then press "Agree and Continue".
+
+You should be redirected to the Console Home screen.
+
+
+## Set up billing
+
+To be able to access the resources required to create a virtual machine, we need to provide billing information, including credit card details. New accounts are given $300 credit to use for some cloud services, however we will need to upgrade to a paid account so that we can use GPUs for Elpis.
+
+On the home screen, click "Billing" in the left hand navigation menu, then click "Add Billing Account". This will open a short, two-step form to provide credit card details and some address information. When this is complete, your account will be on a Free plan.
+
+To use GPUs, we will need to upgrade the account to a paid service. Click the Billing menu item in the Nav menu, and click the "Upgrade" button, then "Activate". These steps seem to change often. For up-to-date information, check the Google documentation for [upgrading a free account](https://cloud.google.com/free/docs/free-cloud-features#to_upgrade_your_account). 
+
+
+## Select APIs
+
+The main resources we will use are Compute Engine Virtual Machines. To get access to Compute Engine we need to enable Compute Engine API, which is a code library that provides access to the Compute Engine. Open the Navigation menu and click "Compute Engine", then "Enable".
```

Comment:
```suggestion
The main resources we will use are Compute Engine Virtual Machines. To get access to Compute Engine we need to enable Compute Engine API, which provides access to the Compute Engine. Open the Navigation menu and click "Compute Engine", then "Enable".
```

Reply:
Aha, I was trying to be more informative (what's an API?) but I suppose that would just start more questions. 

changed in the next release